src/templates/index.html

```diff
@@ -1,22 +1,92 @@
 {{define "/"}}
-	<html lang="en">
-	<head>
-		<meta charset="UTF-8">
-		<title>Welcome!</title>
-	</head>
-
-	<body>
-	<h1>Welcome to the Life Calendar</h1>
-	<p>
-		Unfortunately, this page is empty at the moment. Try the following pages: <a href="/login">/login</a>,
-		<a href="/logout">/logout</a>, <a href="/api">/api</a>/...
-		<br>
-		Currently you are
-		{{if not .}}
-		not
-		{{end}}
-		logged in
-	</p>
-	</body>
-	</html>
+<html lang="en">
+<title>Welcome to Life Calender!</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<style>
+    .w3-sidebar a {font-family: "Roboto", sans-serif}
+    body,h1,h2,h3,h4,h5,h6,.w3-wide {font-family: "Montserrat", sans-serif;}
+    .square {
+        height: 70px;
+        width: 70px;
+        background-color: #0b6;
+    }
+</style>
+
+<body class="w3-content" style="max-width:1200px">
+{{if not .}}
+    Unfortunately, you are not logged in now :(<br>
+    <a href="/login">Log in!</a>
+{{else}}
+    <!-- Sidebar/menu -->
+    <nav class="w3-sidebar w3-bar-block w3-white w3-collapse w3-top" style="z-index:3;width:265px" id="mySidebar">
+        <div class="w3-container w3-display-container w3-padding-16">
+            <i onclick="w3_close()" class="fa fa-remove w3-hide-large w3-button w3-display-topright"></i>
+            <h3 class="w3-wide"><b>Life Calendar</b></h3>
+        </div>
+        <div class="w3-padding-64 w3-large w3-text-grey" style="font-weight:bold">
```

Comment:
Not sure, how serious is this, but it should instead be
```suggestion
        <div class="w3-padding-64 w3-large w3-text-grey" style="font-weight:bold;">
```
Please, note that this mistake also appears in many lines around. If it should be fixed, it should be fixed everywhere